python/test/test_api.py

```diff
@@ -384,5 +452,3 @@ def test_basic_inference(self):
         ):
             fp16_output = numpy.from_dlpack(response.outputs["fp16_output"])
             numpy.testing.assert_array_equal(fp16_input, fp16_output)
-
-        server.stop()
```

Comment:
question: why is stop removed? Is the server automatically destructed if object goes out of scope? If so, you might need explicit `del` otherwise you are relying on garbage collector behaviour which may not happen immediately.

Reply:
server does get destroyed and stop called when it goes out of scope - garbage collector would only come into picture if there is a dangling reference - so don't think explicit del will alter behavior in this case -